bash_completion

```diff
@@ -987,13 +987,27 @@ _comp_variable_assignments()
 #     -o XSPEC    Passed to _filedir as first arg for other output redirections
 #     -i XSPEC    Passed to _filedir as first arg for stdin redirections
 #     -s          Split long options with _split_longopt, implies -n =
+# @param $1...$3 args Original arguments specified to the completion function.
+#                     The first argument $1 is command name.  The second
+#                     argument $2 is the string before the cursor in the
+#                     current word.  The third argument $3 is the previous
+#                     word.
+# @var[out] cur       Reconstructed current word
+# @var[out] prev      Reconstructed previous word
+# @var[out] words     Reconstructed words
+# @var[out] cword     Current word index in `words`
+# @var[out] comp_args Original arguments specified to the completion function
```

Comment:
I have mixed feelings about this name: it's nice and concise, but it somehow slightly annoys me that the name does not make it crystal clear that these are the _original_ completion arguments. Dunno what to do about it if anything, not a huge fan of `comp_orig_args` (or `orig_args` or the same with `original` instead of `orig`)...

Reply:
It was actually the same for me. I also thought of different names such as `original_args`, `comp_original_args`, `old_args`, `args`, etc., but some seemed too long, and the other seemed too general. I thought `prev` and `words` are already general names so initially thought of using `args`, but it turned out `args` are already widely used by completion functions for other purposes. If we wouldn't use `comp_args`, my second candidate is `original_args`.

Reply:
Ok, let's go with `comp_args`.